app/javascript/src/AuthenticationProviders/components/EnableSSOSwitch.tsx

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable react/jsx-curly-newline */
+import {
+  Alert,
+  Spinner,
+  Stack,
+  StackItem,
+  Switch
+} from '@patternfly/react-core'
+
+import type { SwitchProps } from '@patternfly/react-core'
+import type { FunctionComponent } from 'react'
+
+interface Props extends Required<Pick<SwitchProps, 'isChecked' | 'isDisabled'>> {
+  onChange: (checked: boolean) => void;
+  isLoading: boolean;
+}
+
+const EnableSSOSwitch: FunctionComponent<Props> = ({
+  onChange,
+  isChecked,
+  isDisabled,
+  isLoading
+}) => {
+  return (
+    <Stack hasGutter>
+      <StackItem>
+        <Switch
+          id="settings_enforce_sso"
+          isChecked={isChecked}
+          isDisabled={isLoading || isDisabled}
+          label={(
+            <>
+                Enable SSO for all users of this account {isLoading && <Spinner isInline isSVG />}
+            </>
+          )}
+          name="settings[enforce_sso]"
+          onChange={onChange}
+        />
+      </StackItem>
+      {isDisabled && (
+        <StackItem>
+          <Alert
+            isInline
+            title="To enable SSO, make sure you have a published SSO integration that was tested
```

Comment:
`To enforce SSO`

Reply:
It's actually "enable" now, according to the docs team microcopy review.